### ISSUE TITLE
commands editor creates commands.pak with increasing size then takes long or fails to re-open

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#212]
I started to ask about this on discord other night and was going to not report it as a one time thing. 
Having a problem with commands.pak for sci_hospitallower.  It hangs the commands editor while loading 
and it seems to be related that it's making larger and larger files.  Here's a screenshot of the folder with backups.   
For reference the commands.pak at the top is the base size and loads into commands editor no time.  
The rest all take minutes. Long as 20-30m with the window displaying not responding.

![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/de00b9b2-9cdd-42da-bbe1-508a73767208)

In the game they all function as intended. Just doing this to commands editor.  It was suggested that a corrupted 
AI install might be at fault. I tried the files on the EGS version and same symptoms.  

This might be a one time thing only for this map file, but before I go deleting these files I want to make sure
 you don't want to examine for any issue that has caused the hang ups and size increase.

**Comment by ghramsey** [comment]
I needed to open one of the files that was taking a long time to open and just let it take it's time.
It loaded and I've copied off my settings to a new file.  no problems there but I've saved the "bad" file.
![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/dd2896b1-a993-49ad-8f4d-3e165a2bf8b9)
It grew in size again. Doubled to 24.1k.

I'll hold on to this as a sample of whatever this issue is.

**Comment by MattFiler** [comment]
Interesting - please could you link the PAK that doubled when you saved?

**Comment by ghramsey** [comment]
I had a feeling you might want them. 

I put them into a cloud folder. Since the smaller files grew to the larger ones.

Let me know if there are any issues with the link.  It should be a public DL/UL folder:
https://u.pcloud.link/publink/show?code=kZz0pIVZlmAppEx6w8VfbN73il2hy8aBb9wy

**Comment by MattFiler** [comment]
I figured out what was causing this, and it should now be fixed in staging. Your broken PAKs will continue to be broken though unfortunately, so you'll have to revert back to before this issue started happening - sorry!

**Comment by ghramsey** [comment]
Yeah.  I've already reverted to an older backup and recreated my work.

**Comment by ghramsey** [comment]
I seem to have figured out what was causing this issue and fixed one of the broken commands.pak files.

There was a corrupted entry in ayz\levels\sevastopol\science\hospital\maindeck\staff\staff_section\
The entry labeled Waits To Morley which is a Terminal Content function.  It had the verbatim text from the T0001 terminal 
file shown in the addional_info field:

 "Dr. Morley, I don‘t care what you think you need to do. I want you out of San Cristobal NOW.

You know what we‘ve set up in there, and I‘m truly sorry for what it means for the patients who can‘t be moved, but this is the only way.

There will be no further warnings."
-

I've pasted that here with the white space included. When I used the AI PAK differentiation program it showed the issue 
plainly. All that white space was showing as corrupted characters.    This was hanging the listbox preventing my pasting the entry A1_T0001_TEX_2736 in place of the plain text nor was the dropdown window allowing selection it opened and closed right away.  The solution was to remove the additional_info parameter and recreated it then paste in the correct Terminal text data reference.

This seemed to fix at least part of the issue.  The file no longer hangs the commands editor while loading instead taking only moments.  I shall see if I can open the other broken files and make this fix to those too.   

I know you said you'd fixed a related bug too, but for information sake I'm adding this here.

**Comment by ghramsey** [comment]
I confirmed the fixed on one of those 10kb files.  Took 20m to load, but once it did I made the edit outlined above and the file size deflated from 10k+ to 8.2k.

**Comment by MattFiler** [comment]
Hmm, yeah this was the issue I identified with your previous PAK and what I thought I had fixed on staging. Are you using the latest staging and this is still occuring? Did you start with a clean PAK again, or were you using one that was previously edited?

**Comment by ghramsey** [comment]
It has recurred if I use the PAK file I have edited over months. I've reverted to PAK files from March and before which 
were working before.  It still recurs if I save a few times. Now that I know what is at fault though I can fix it before it 
balloons to the point of hanging up the editor.  It's odd it only hangs the editor, not in the game.

I've tried to begin anew with a fresh PAK file, but I have made so many changes I can't recreate some of them 
or I lost track of what changes I made.    I have used these older files as templates to create a frash PAK, but there is still 
not real way to find all the edits. Even the AI PAK differentiator is not finding everything.  There are a bunch of save stations
only showing that an entity has been created but not showing the details such as transform. IDK if that is a bug in that 
program or something else but it makes tracking the edits via file compare harder.
